VedaZilla.user.js

```diff
@@ -60,19 +60,61 @@
       VZ_BLACK = "#000000",
       TIMEOUT_HANDLER = 250 /*ms*/;
 
+  // Configuration Page
+  MH_PAGE_HANDLER["MH_Play/Options/Play_o_Interface"] = function(p, l) {
+    // SetVZValue('HOME_BUTTON', 0);
+      let config = $('<fieldset>').addClass("mh_tdtitre");
+      let legend = $('<legend>').text('VedaZilla Configuration').addClass("mh_tdtitre").css({'border': 'thin solid black', width: "80%", padding: "0.3rem 0"});
+      let connexionButton = $('<button>').click(function(){
+          console.log('Before ' + GetVZValue('HOME_BUTTON'));
+          if(GetVZValue('HOME_BUTTON') == 1){
```

Comment:
Why is this not a '===' (as line 82)? Looks a bit scary.
Also the case on line 96.

Reply:
(I've been had by the separation in two commits, but the point remains, why not use ===)

Reply:
GetVZValue returns a string.

If you want a strict test,

`if(GetVZValue('HOME_BUTTON') === '1'){`